Transporter.IntegrationTests/Tests/Couchbase-MSSQL/CouchbaseToMssqlIntegrationTests.cs

```diff
@@ -0,0 +1,140 @@
+            _fixture = new Fixture();
+            _mockDateTimeProvider = new Mock<IDateTimeProvider>();
+            _mockDateTimeProvider.Setup(m => m.Now).Returns(DateTime.UtcNow.AddMinutes(-5));
+        }
+
+        [Test]
+        public async Task Transporter_ShouldTransportData()
+        {
+            //Arrange
+            await GetRequiredService<ICouchbaseProviderHelper>()
+                .CreateBucketAsync(GetCouchbaseConnectionData(), JobConstants.CouchbaseToMsSqlJob.SourceBucketName);
+            Thread.Sleep(2000);
+            var collection =  await GetRequiredService<IBucketProvider>()
+                .GetBucket(GetCouchbaseConnectionData(), JobConstants.CouchbaseToMsSqlJob.SourceBucketName);
+            await GetRequiredService<ICouchbaseProviderHelper>()
+                .CreatePrimaryIndexAsync(GetCouchbaseConnectionData(), $"`{JobConstants.CouchbaseToMsSqlJob.SourceBucketName}`");
+            
+            await using var connection = new SqlConnection(MssqlConnectionString);
+            connection.Open();
+
+            var id = CreateFixture<long>();
+            var age = CreateFixture<int>();
+            await connection.ExecuteAsync("Create Table dbo.TestTable_Interim (dataSourceName nvarchar(max), lmd DateTime, Id int)");
+            await connection.ExecuteAsync("Create Table dbo.TestTable_Target (Id nvarchar(max), Age int)");
+            var couchbaseCollection =    await collection.DefaultCollectionAsync();
+            var user = _fixture.Build<UserClass>().With(x=> x.Age, age)
+                .With(x=> x.Id, id).Create();
+            await couchbaseCollection.InsertAsync(id.ToString(), user);
+            var configSettings = GetConfigSettings();
+            var configuration = new ConfigurationBuilder()
+                .AddInMemoryCollection(configSettings)
+                .Build();
+            
+            var mockDateTimeProvider = new Mock<IDateTimeProvider>();
+            mockDateTimeProvider.Setup(m => m.Now).Returns(DateTime.UtcNow.AddMinutes(-5));
+            
+            var build = Program.CreateHostBuilder(Array.Empty<string>())
+                .ConfigureAppConfiguration((context, config) =>
+                {
+                    config.AddConfiguration(configuration);
+                })
+                .ConfigureServices((_, services) =>
+                {
+                    services.AddTransient(_ =>_mockDateTimeProvider.Object);
+                }) 
+                .Build();
+            
+            await build.StartAsync();
+            Thread.Sleep(65000);
+            await build.StopAsync();
+
+            //Verify
+            var userFromTargetTable = await connection.QueryFirstAsync<UserClass>("SELECT Id FROM [dbo].[TestTable_Target]");
+            id.Should().Be(userFromTargetTable.Id);
+        }
+
+        private Dictionary<string, string> GetConfigSettings()
+        {
+            var configSettings = new Dictionary<string, string>
+            {
```

Comment:
what do you think about having a JSON file for these configs? Would it be better, or would it have more cons?

Reply:
json files created for job settings like "couchbasetoCouchbase_appsettings.json" for each integration test type.